app/js/directives/yesNoQuestion.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+angular.module('ddsApp').directive('yesNoQuestion', function() {
+    return {
+        restrict: 'E',
+        transclude: true,
+        templateUrl: 'partials/foyer/yes-no-question.html',
+        scope: true,
+        controller: 'yesNoQuestionCtrl',
+        link: function ($scope, $element, $attributes) {
+            $scope.modelString = $attributes.model;
+        }
+    };
+});
+
+angular.module('ddsApp').controller('yesNoQuestionCtrl', function($scope, $transclude, $element, $interpolate) {
+	_.forEach($transclude(), function(elem) {
+		if (elem.localName == 'question') {
+			$element.find('legend').append($interpolate(elem.innerHTML)($scope));
+		} else if (elem.localName == 'help-block') {
+			$element.find('p').append($interpolate(elem.innerHTML)($scope));
+		}
+	});
+
+	// To get the object and the property described by the `model` directive attribute.
+    function parse(modelString) {
+        var re = /(.*)[.](.*)/;
+        var resultRegexp = re.exec(modelString);
+        return {
+            objectToModify: $scope.$eval(resultRegexp[1]),
+            propertyToModify: resultRegexp[2]
```

Comment:
Much cheaper too. Regexes are costly both to compile and execute  😃 

Beware of the behaviour if the given property path doesn't exist, though. You might be interested in checking it either [like this](https://github.com/MattiSG/Watai/blob/679619ed98a86ba52024960aa7816ba1cfe60e66/src/lib/mootools-additions.js#L4-L22) or through `$scope.$eval` (but does it throw if there's an undefined somewhere in the chain?